plugins/backstage-plugin-coder/src/api/CoderClient.ts

```diff
@@ -352,6 +354,23 @@ export class CoderClient implements CoderClientApi {
   };
 }
 
+function appendParamToQuery(
+  query: string | undefined,
+  key: string,
+  value: string,
+): string {
+  const keyValuePair = `param:"${key}=${value}"`;
```

Comment:
Is it worth check that these don't contain empty strings?

Reply:
Oh, good call. Let me fix that really quick